javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -230,6 +230,26 @@ public void shouldLazyReturnNotNull() {
         assertThat(lazy(() -> 1)).isNotNull();
     }
 
+    @Test
+    public void shouldOptionReturnNotNull() {
+        assertThat(option(1)).isNotNull();
```

Comment:
Likewise to Scala

* option(null) should be None
* some(null) should be Some(null)